README.md

```diff
@@ -6,41 +6,47 @@
 1. [**Overview**](#Overview)
 2. [**How to add it to project**](#HowToAddItToProject)
 3. [**How to use it**](#HowToUseIt)
-4. [**Events versioning**](#EventsVersioning)
-4. [**Databases**](#Databases)
-6. [**Authors**](#Authors)
-7. [**License**](#License)
+4. [**Optimistic locking**](#OptimisticLocking)
+5. [**Events versioning strategies**](#EventsVersioningStrategies)
+6. [**Databases**](#Databases)
+7. [**Authors**](#Authors)
+8. [**License**](#License)
 
 ## Overview <a name="Overview"></a>
 
 If you want to use event sourcing together with java and sql database this library is for you.
 In additional to its primary goal, which is event persistence, it also:
 * helps to deal with DDD aggregates (supports aggregate recreation from events)
 * supports hexagonal architecture approach (does not force your events and aggregates to extend library-specific classes)
-* supports events versioning
-* COMING SOON: supports optimistic locking
-* COMING SOON: supports streams versioning
-* TODO: supports pessimistic locking
-* TODO: supports copy&replace approach
+* supports multiple [strategies of events versioning](#EventsVersioningStrategies)
+* supports [optimistic locking](#OptimisticLocking)
+* COMING SOON: supports pessimistic locking
+* COMING SOON: supports copy&replace approach
 
 ## How to add it to project <a name="HowToAddItToProject"></a>
 
 ### Add dependency
 
-If you are using gradle add it to build.gradle:
+If you are using gradle add this to build.gradle:
 ```groovy
 dependencies {
-    implementation "com.hltech:sql-event-store-4j:version"
+    implementation "com.hltech:sql-event-store-4j:0.0.7"
```

Comment:
maybe lets stick to 'version', we wont have to upgrade this piece of readme on each release